allConfigsRunner.py

```diff
@@ -1,17 +1,34 @@
-import time, random, constants, subprocess
+import time, random, constants, subprocess, os
 from pathlib import Path
 
+
 def main():
     configs_path = Path("configs")
     config_files = sorted(configs_path.glob('*_config.py'))
 
     for config_file in config_files:
         print(f"Starting LinkedIn application with configuration: {config_file.name}")
         # Copy the current config file to config.py
-        subprocess.run(["cp", "-f", str(config_file), "config.py"], check=True)
+
+        # subprocess.run(["cp", "-f", str(config_file), "config.py"], check=True)
+        # Check if the operating system is Windows
+        if os.name == 'nt':  # 'nt' indicates a Windows environment
+            # Use the 'copy' command in Windows
+            subprocess.run(["copy", str(config_file), "config.py"], shell=True, check=True)
+        else:
+            # Use the 'cp' command for Unix/Linux
+            subprocess.run(["cp", "-f", str(config_file), "config.py"], check=True)
+
 
         # Run the LinkedIn Easy Apply bot
-        subprocess.run(["python3", "runner.py"], check=True)
+        # subprocess.run(["python3", "runner.py"], check=True)
+        # Check if the operating system is Windows
+        if os.name == 'nt':  # 'nt' indicates a Windows environment
+            # Use 'python' for Windows
+            subprocess.run(["python", "runner.py"], check=True)
+        else:
+            # Use 'python3' for Unix/Linux
+            subprocess.run(["python3", "runner.py"], check=True)
```

Comment:
I would rather use a try / catch if possible in this context because this is more a question how python is defined in your PATH environmental variable. Someone might have "py", "python", "python3", etc. 

You can test it by first trying to run "python3", catching and then trying to run "python" because currently, the Docker container is installing and using python3.

Btw, this is not necessary for this PR, but I would suggest you installing Docker if you don't have it already and try running the code in a container. You'll thank me later :)